src/helm/proxy/clients/huggingface_model_registry.py

```diff
@@ -0,0 +1,81 @@
+from typing import Dict, Optional
+from dataclasses import dataclass
+import re
+from helm.common.hierarchical_logger import hlog
+from helm.proxy.models import (
+    Model,
+    MODEL_NAME_TO_MODEL,
+    TEXT_MODEL_TAG,
+    FULL_FUNCTIONALITY_TEXT_MODEL_TAG,
+)
+
+
+@dataclass(frozen=True)
+class HuggingFaceModelConfig:
+    namespace: Optional[str]
+    """Name of the group or user that owns the model. e.g. 'stanford-crfm'"""
+
+    model_name: str
+    """Name of the model. Does not include the namespace. e.g. 'pubmed-gpt2'"""
```

Comment:
maybe choose a example name now...

Reply:
`BioMedLM` and `BioMedLM#main` (it doesn't have any other revisions lol